lib/tasks/swaggerize.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+
+namespace :rswag do
+  namespace :specs do
+    desc 'Generate Swagger JSON files from integration specs'
+    RSpec::Core::RakeTask.new('swaggerize') do |t|
+      t.pattern = ENV.fetch(
+        'PATTERN',
+        'spec/integration/**/*_spec.rb'
+      )
+      t.rspec_opts = ['--format Rswag::Specs::SwaggerFormatter', '--order defined']
+    end
+  end
+end
+
+task rswag: ['rswag:specs:swaggerize']
```

Comment:
Is this documented somewhere in our README?

Reply:
It is: https://github.com/RedHatInsights/compliance-backend/blame/5338b2402ac5cf645dcdfb6d211fa47f00f20f4d/README.md#L357